staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers.go

```diff
@@ -109,9 +109,6 @@ func WriteObjectNegotiated(ctx request.Context, s runtime.NegotiatedSerializer,
 		audit.LogResponseObject(ae, object, gv, s)
 	}
 
-	w.Header().Set("Content-Type", serializer.MediaType)
```

Comment:
I see. I looked into the golang/x variant and saw the panic. Looks like the "official" lib only warns. :+1: